app/src/main/java/com/example/crumbtrail/ComposeActivity.java

```diff
@@ -80,9 +82,21 @@ protected void onCreate(Bundle savedInstanceState) {
                     Toasty.error(ComposeActivity.this, "Error while saving!", Toasty.LENGTH_SHORT).show();
                 }
                 Log.i(TAG, "Post save was successful!");
+                notifyUser();
             });
             progressBar.setVisibility(View.GONE);
             finish();
         });
     }
+
+    private void notifyUser() {
```

Comment:
Is the purpose of this to let the user currently leaving a review to then also get a push notif once that successfully goes through? Or are you hoping to push notify other users that have some kind of connection to this review (like they follow the user or the item)?

Reply:
I'm hoping to notify users that follow this item (This is still a WIP as I'm not quite sure how to notify users that have liked a review on a food item)